### PR TITLE
Climatic Methods -Water Balance

### PR DESCRIPTION
**Pull request by aduut**
Water Balance dialog with all parameters.

### CODE REVIEW

**Comment by volloholic** [comment]
Abib this branch is not yet ready to merge please see the comments in the code.

File: instat/dlgWaterBalance.vb [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
-
-        strScript = strScript & ", col_name=" & Chr(34) & txtColumnName.Text & Chr(34)
-        strScript = strScript & ")"
-        frmMain.clsRInterface.climateObject()
-        frmMain.clsRInterface.RunScript(strScript, False)
+    Private Sub UcrButtons1_Load(sender As Object, e As EventArgs) Handles ucrBase.Load
+        autoTranslate(Me)
+        ucrBase.clsRsyntax.SetFunction("climate_obj$display_water_balance()")
+        ucrBase.clsRsyntax.iCallType = 1
         frmMain.FillData("climate_obj$climate_data_objects[[1]]$data")
```
Comment (volloholic) [inline instat/dlgWaterBalance.vb]: This does not belong in load.

File: instat/dlgWaterBalance.vb [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
 
-        If chkPrintTable.Checked Then
-            strScript = strScript & ", print_tables=" & chkPrintTable.Checked.ToString().ToUpper
-        End If
-
-        strScript = strScript & ", col_name=" & Chr(34) & txtColumnName.Text & Chr(34)
-        strScript = strScript & ")"
-        frmMain.clsRInterface.climateObject()
-        frmMain.clsRInterface.RunScript(strScript, False)
+    Private Sub UcrButtons1_Load(sender As Object, e As EventArgs) Handles ucrBase.Load
```
Comment (volloholic) [inline instat/dlgWaterBalance.vb]: rename UcrButtons1 to ucrBase

File: instat/frmMain.vb [hunk 3]
```diff
@@ -420,10 +420,14 @@ Public Class frmMain
     End Sub
 
     Private Sub OneSampleToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles OneSampleToolStripMenuItem.Click
-        dlgOneSample.ShowDialog()
+        'dlgOneSample.ShowDialog()
     End Sub
```
Comment (volloholic) [inline instat/frmMain.vb]: Are these changes you made? I don't believe they should be commited as you are not working on onesample or frequency

**Comment by maxwellfundi** [comment]
This is now redone, It can be closed now.

**Pull request closed without merging**